ibm/mas_devops/roles/cp4d_install/defaults/main.yml

```diff
@@ -10,12 +10,9 @@ cpd_registry_user: cp
 # This is only used for cpd35, we have to use cp4d_hack_worker_nodes to set up the global image pull secret needed by cpd40
 cpd_entitlement_key: "{{ lookup('env', 'CPD_ENTITLEMENT_KEY') }}"
 
-# Both CP4D storage classes have the same storage requirements as per recommendation from cpd support team
-# so we set storageClass and zenCoreMetadbStorageClass to "{{ cpd_storage_class }}"
-# https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=requirements-storage
-#
 # If this changes in the future we will expose a seperate variable for each.
 cpd_storage_class: "{{ lookup('env', 'CPD_STORAGE_CLASS') }}"
+cpd_metadb_block_storage_class: "{{ lookup('env', 'CPD_METADB_BLOCK_STORAGE_CLASS') }}"
```

Comment:
Can we have this new env var default to whatever CPD_STORAGE_CLASS is if CPD_METADB_BLOCK_STORAGE_CLASS is not set? Then at least it is backwards compatible. It might not be the recommended for a production CP4D but it would still be a supported setup I believe.

Reply:
I can include a default value for both vars but it would only be applicable for ROKS, for Fyre/Quickburn it woudn't work as they have different storage classes.
I think that's the reason why we didn't specified default values and instead added a check to ensure both are set in the `main.yml` task.

Reply:
@andrercm For the default value I meant if CPD_METADB_BLOCK_STORAGE_CLASS is not set then lookup CPD_STORAGE_CLASS instead, rather than set a specific storage class default. This way users that have just CPD_STORAGE_CLASS set will still continue to work but can set CPD_METADB_BLOCK_STORAGE_CLASS if required.